rollup-bridge-contracts/setup.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Function to check if a command exists
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+# Check if .env file exists
+if [ -f .env ]; then
+    echo "Warning: .env file already exists. Skipping copy from .env.example."
+else
+    echo "Copying .env.example to .env..."
+    cp .env.example .env
+fi
+
+# Install npm dependencies
+echo "Installing npm dependencies..."
+npm install
+
+# Run Hardhat clean and compile
+echo "Running Hardhat clean and compile..."
+npx hardhat clean
+npx hardhat compile
+
+# Install Forge dependencies
+if command_exists forge; then
+    echo "Installing Forge dependencies..."
+
+    # Check and install foundry-rs/forge-std if not already installed
+    if [ ! -d "lib/forge-std" ]; then
+        echo "Installing foundry-rs/forge-std..."
+        forge install foundry-rs/forge-std --no-commit
+    else
+        echo "foundry-rs/forge-std already installed. Skipping."
+    fi
+
+    # Check and install transmissions11/solmate if not already installed
+    if [ ! -d "lib/solmate" ]; then
+        echo "Installing transmissions11/solmate..."
+        forge install transmissions11/solmate --no-commit
+    else
+        echo "transmissions11/solmate already installed. Skipping."
+    fi
+
+    # Check and install dapphub/ds-test if not already installed
+    if [ ! -d "lib/ds-test" ]; then
+        echo "Installing dapphub/ds-test..."
+        forge install dapphub/ds-test --no-commit
+    else
+        echo "dapphub/ds-test already installed. Skipping."
+    fi
+
+    # Remove .gitmodules file if it exists
+    if [ -f .gitmodules ]; then
+        echo "Removing .gitmodules file..."
+        rm .gitmodules
+    fi
+
+    # Remove .gitmodules file if it exists
```

Comment:
duplicate?
No `gitmodules` in PR, these lines still actual?

Reply:
forge install is used to install essential foundry dependencies for solidity smart contracts
the f`orge install` by default creates `.gitmodules` file
so added this in the setup script
and same is made as part of `.gitignore` file to avoid commit to project 

at moment all forge dependencies are installed only via `forge install`